src/components/Calculator.js

```diff
@@ -0,0 +1,173 @@
+import '../styles/Calculator.css';
+import React from 'react';
+
+/* eslint-disable-next-line react/prefer-stateless-function */
```

Comment:
- [ ] Let's make sure you don't disable linter rules as these are here to help us write clean code.

Reply:
Hello @arslanbisharat  thank you for your feedback, but as per the requirement of this milestone all components must `class based components`, hence the reason I need to disable the particular lint
Kindly advise on that, if not I will be forced to use `stateless function` which is not according to the project requirement